main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -514,7 +535,7 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
     */
   def build(flix: Flix): Validation[CompilationResult, BootstrapError] = {
     // Configure a new Flix object.
-    val newOptions = flix.options.copy(output = Some(Bootstrap.getBuildDirectory(projectPath)))
+    val newOptions = flix.options.copy(outputJvm = true, outputPath = Bootstrap.getBuildDirectory(projectPath))
```

Comment:
Don't know if we actually want `outputJvm = true` here, the previous code just didn't have another option.